packages/office-ui-fabric-react/src/components/Modal/Modal.types.ts

```diff
@@ -63,6 +63,13 @@ export interface IModalProps extends React.ClassAttributes<ModalBase>, IWithResp
    */
   isBlocking?: boolean;
 
+  /**
+   * Whether the dialog should be sticky (e.g. not dismiss when focusing/clicking outside of the dialog).
+   * if true: isBlocking is ignored, there will be no overlay (isDarkOverlay is ignored),
+   * isClickableOutsideFocusTrap is true, and forceFocusInsideTrap is false
+   */
+  isSticky?: boolean;
```

Comment:
I think I would prefer `isModeless` but that also highlights another issue: `Modal` should inherently be modal. I think a modeless modal is just a dialog. Should `Modal` support this prop? Can `Dialog` be used instead?

But now looking at `Modal` and `Dialog` implementations I see that `Dialog` actually uses `Modal`. I would have expected that to be the other way around. A `Modal` is a specialized blocking version of `Dialog`. I would have expected `Modal` to use `Dialog`. 😕 

TL;DR: I guess my conclusion is that I would prefer `isModeless` wording as more standardized and descriptive.

Reply:
Yeah, the same thoughts went through my head (hence why I called it sticky instead of modeless). I'll update it to modeless